services/coincap/coincap-changepercent24hr.service.js

```diff
@@ -0,0 +1,42 @@
+import Joi from 'joi'
+import { floorCount } from '../color-formatters.js'
+import BaseCoincapService from './coincap-base.js'
+
+const schema = Joi.object({
+  data: Joi.object({
+    changePercent24Hr: Joi.string().required(),
+    name: Joi.string().required(),
+  }).required(),
+}).required()
+
+export default class CoincapChangePercent24HrUsd extends BaseCoincapService {
+  static route = { base: 'coincap/changePercent24Hr', pattern: ':assetId' }
+
+  static examples = [
+    {
+      title: 'Coincap (Change Percent 24Hr)',
+      namedParams: { assetId: 'bitcoin' },
+      staticPreview: this.render({
+        asset: { changePercent24Hr: '2.0670573674501840"' },
+      }),
+      keywords: ['bitcoin', 'crypto', 'cryptocurrency'],
+    },
+  ]
+
+  static percentFormat(changePercent24Hr) {
+    return `${parseInt(changePercent24Hr).toFixed(2)}%`
+  }
+
+  static render({ asset }) {
+    return {
+      label: asset.name,
+      message: this.percentFormat(asset.changePercent24Hr),
+      color: floorCount(asset.changePercent24Hr),
```

Comment:
What are we trying to achieve by using `floorCount()` with no params for formatting here?
Tbh, I'd be inclined to make this an informational blue badge too rather than try to use colour to convey meaning here. It seems like it would be hard to definitively even say whether stability or volatility is 'good' or 'bad' in all cases.

Reply:
It is a convention in finance, market information with negative percentage variation be red and positive green. Including coincap.

Reply:
Right yes I see where you are going with this now :smile: